docs/node/run-your-node/prerequisites/hardware-recommendations.md

```diff
@@ -83,8 +83,8 @@ that use a trusted execution environment (TEE).
   * Recommended: 8 GB of ECC RAM
 
 * All ParaTime compute or client nodes:
```

Comment:
```suggestion
* Emerald, Cipher or Sapphire compute or client node (figures below are per ParaTime):
```

Reply:
If we start listing ParaTimes explicitly, then we should probably list them all over the place?

Reply:
Can you be more specific? CPU and storage requirements are already specific for each ParaTime. The existing `All ParaTime compute or client nodes` in the Memory section is misleading, because you can always [build your own ParaTime](https://docs.oasis.io/paratime/) where memory requirements could be very small (e.g. the key manager ParaTime).

Reply:
You are right.